vsix/Component/source.extension.vsixmanifest

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.Windows.CppWinRT" Version="|%CurrentProject%;GetCppWinRTVersion|" Language="en-US" Publisher="Microsoft" />
+    <PackageId>Microsoft.Windows.CppWinRT</PackageId>
+    <DisplayName>C++/WinRT</DisplayName>
+    <Description xml:space="preserve">Tools for authoring and consuming Windows Runtime classes in standard C++.</Description>
+    <MoreInfo>https://go.microsoft.com/fwlink/?linkid=869449</MoreInfo>
+    <License>LICENSE</License>
+    <GettingStartedGuide>https://docs.microsoft.com/windows/uwp/cpp-and-winrt-apis/intro-to-using-cpp-with-winrt</GettingStartedGuide>
+    <ReleaseNotes>https://docs.microsoft.com/windows/uwp/cpp-and-winrt-apis/faq</ReleaseNotes>
+    <Icon>cppwinrt.ico</Icon>
+    <PreviewImage>cppwinrt.png</PreviewImage>
+    <Tags>WinRT, C++, cppwinrt, native</Tags>
+  </Metadata>
+  <Installation AllUsers="true">
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
```

Comment:
just curious, will this line, as written, block on 17.1, 17.2, etc.? If so, is that intentional?

Reply:
This line will actually block starting on 17.0 and up. I didn't write this line originally, but I imagine it was intentional, with the thinking being that we'd want to actually test this extension on 17.0 first. This seems like a good decision, especially in light of some upcoming changes to VS 17.